values.yaml

```diff
@@ -635,6 +635,11 @@ concourse:
     newrelic:
       enabled: false
 
+      ## New Relic Insights API base Url
+      ## For more info:
+      ## https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/introduction-eu-region-data-center#endpoints
+      url: https://insights-collector.newrelic.com
```

Comment:
In order to reduce duplication and more importantly drift from the concourse binary, we're intentionally not defining the values with their defaults from the binary.

Could you remove the default from here? 
It would be helpful to add it to the comment as an example.

Lastly, sorry this is not obvious or documented anywhere at the moment.

Reply:
Thank you for the review